blocks/library/embed/index.js

```diff
@@ -181,6 +189,21 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 				const parsedUrl = parse( url );
 				const cannotPreview = includes( HOSTS_NO_PREVIEWS, parsedUrl.host.replace( /^www\./, '' ) );
 				const iframeTitle = sprintf( __( 'Embedded content from %s' ), parsedUrl.host );
+				const embedWrapper = 'wp-embed' !== type ? (
```

Comment:
Minor: In a condition or ternary with both `if` and `else` parts, I tend to encourage the positive form as the first case, since "if _positive_ else _negative_" reads more naturally than "if _negative_ else _positive_".